contrib/snap/snapcraft.yaml

```diff
@@ -0,0 +1,236 @@
+name: fwupdtool
+version-script: cat $SNAPCRAFT_STAGE/version
+version: 'daily'
+summary: fwupd
+description: |
+  fwupd.
```

Comment:
doesn't seem very verbose...

Reply:
Oh yeah, I forgot to update that.  Good catch, just fixed.